public/app/percona/dbaas/components/Kubernetes/PortalK8sFreeClusterPromotingMessage/PortalK8sFreeClusterPromotingMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Alert, useStyles2 } from '@grafana/ui';
+import { getStyles } from './PortalK8sFreeClusterPromotingMessage.styles';
+
+export const PortalK8sFreeClusterPromotingMessage = () => {
+  const styles = useStyles2(getStyles);
+  return (
+    <div className={styles.alertWrapper}>
```

Comment:
The `Alert` component allows to pass `className`, so I think we can avoid an extra div here